a2i-code-For-Govstack/sp-service-solutions-api
# [3] – New endpoint to get all public surveys

## Description

This PR introduces a new endpoint to get a list of all public surveys (links).

<img src="https://github.com/user-attachments/assets/2f3f25c5-8d61-42d6-a4e8-786e08d51580" width="70%">

Note: The domain here is dynamically fetched.


 ### How to test
- Make a `GET` request to `/api/v1/data/voter/open-forms`.
- You should be seeing the expected response.

Signed by Divij Sharma

CC @Sanchay-T
